cli/submit.py

```diff
@@ -2,85 +2,32 @@
-        if response.status_code == 200:
-            logger.info("LSRI authentication successful!")
-            response_json = json.loads(response.text)
-            logger.info("Submission ID {} received!!".format(response_json["submissionId"]))
-            self.upload_submission(response_json["submissionId"], response_json["uploadUrl"])
-        else:
-            raise RuntimeError("Could not perform LSRI Authentication! Please try running this script again.")
+    def __init__(self, submission_initiate_url=SUBMISSION_INITIATE_URL):
+        self.auth = get_auth()
+        self.submission_initiate_url = submission_initiate_url
 
     # TODO
     def upload_submission(self, submission_id, submission_upload_url):
         pass
 
-    @staticmethod
-    def _get_webin_credentials():
-        username = input("Enter your ENA Webin username: ")
-        password = getpass("Enter your ENA Webin password: ")
-        return username, password
-
-    def submit_with_webin_auth(self, username, password):
-        logger.info("Proceeding with ENA Webin authentication...")
-
-        headers = {"accept": "*/*", "Content-Type": "application/json"}
-        data = {"authRealms": ["ENA"], "username": username, "password": password}
-        response = requests.post(self.ena_auth_url, headers=headers, data=json.dumps(data))
-
-        if response.status_code == 200:
-            logger.info("Webin authentication successful!")
-            webin_token = response.text
-            response = requests.post(self.submission_initiate_webin_url,
-                                     headers={'Accept': 'application/hal+json',
-                                              'Authorization': 'Bearer ' + webin_token})
-            response_json = json.loads(response.text)
-            logger.info("Submission ID {} received!!".format(response_json["submissionId"]))
-            self.upload_submission(response_json["submissionId"], response_json["uploadUrl"])
-        else:
-            logger.error("Authentication failed!")
-
-    def auth_prompt(self):
-        print("Choose an authentication method:")
-        print("1. ENA Webin")
-        print("2. LSRI")
-
-        choice = int(input("Enter the number corresponding to your choice: "))
-
-        if choice == 1:
-            webin_username, webin_password = StudySubmitter._get_webin_credentials()
-            self.submit_with_webin_auth(webin_username, webin_password)
-        elif choice == 2:
-            self.submit_with_lsri_auth()
-        else:
-            logger.error("Invalid choice! Try again!")
-            self.auth_prompt()
+    def submit(self):
+        response = requests.post(self.submission_initiate_url,
+                                 headers={'Accept': 'application/hal+json',
+                                          'Authorization': 'Bearer ' + self.auth.token()})
+        response_json = json.loads(response.text)
```

Comment:
Is there a reason we can't use `response.json()` here?

Reply:
Also we should check `response.status_code == 200` or use `response.raise_for_status()` first.

Reply:
Done